delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/serialization/EventSerializer.scala

```diff
@@ -0,0 +1,112 @@
+package ch.epfl.bluebrain.nexus.delta.serialization
```

Comment:
It should be more in the service module, where we can also test it at the same time as the module impls no ?

Reply:
yep